pkg/scheduler/actions/preempt/preempt.go

```diff
@@ -197,6 +197,7 @@ func preempt(
 	ssn *framework.Session,
 	stmt *framework.Statement,
 	preemptor *api.TaskInfo,
+	job *api.JobInfo,
```

Comment:
@wpeng102 Are you adding a preemptorJob parameter, if it is, it's better to get the preemptorJob from the preemptor without adding a new parameter.

Reply:
Address comments in new commit